src/index.html

```diff
@@ -9,9 +9,216 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
+    <script type="text/javascript" src="./main.js"></script>
   </head>
+
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      target="_blank"
+    >
+      <fieldset class="personalInfo form-set">
+        <legend>Personal information</legend>
+
+        <div class="form-field">
+          <label for="surname">Surname:</label>
+
+          <input
+            type="text"
+            id="surname"
+            name="surname"
+            autocomplete="off"
+          >
+        </div>
```

Comment:
You can wrap `input` with `label` and set styles for label.
 In this case you don't need to use for/id as well
```suggestion
        <label class="form-field">
          Surname:
          <input
            type="text"
            name="surname"
            autocomplete="off"
          >
        </label>
```

Reply:
Thanks for answer. I fixed all issues. Unused classes removed, margins fixed, 'div' replaced and placeholder as well. Your comments were very helpful.